src/main/java/io/github/balconyseats/vertx/helper/kafka/KafkaHelper.java

```diff
@@ -0,0 +1,170 @@
+ * kafka:
+ *   enabled: true
+ *   consumers:
+ *     commonConfig:
+ *       key.deserializer : 'org.apache.kafka.common.serialization.StringDeserializer'
+ *       value.deserializer : 'org.apache.kafka.common.serialization.StringDeserializer'
+ *     foo:
+ *       client.id: foo
+ *       topic: foo_topic
+ *       group.id: foo
+ *     bar:
+ *       client.id: bar
+ *       topic: bar_topic
+ *       group.id: bar
+ *   producers:
+ *     commonConfig:
+ *       key.serializer : 'org.apache.kafka.common.serialization.StringSerializer'
+ *       value.serializer : 'org.apache.kafka.common.serialization.StringSerializer'
+ *     foo:
+ *       client.id: foo
+ *       topic: foo
+ *     bar:
+ *       client.id: bar
+ *       topic: bar
+ *   config:
+ *     bootstrap.servers: 'server-1:9092,server-2:9092'
+ *     security.protocol : 'SSL'
+ *     ssl.enabled.protocols: 'TLSv1.2,TLSv1.1,TLSv1'
+ *     ssl.keystore.location: 'keystore.jks'
+ *     ssl.keystore.password: 'changeit'
+ *     ssl.keystore.type: 'JKS'
+ *     ssl.truststore.location: 'truststore.jks'
+ *     ssl.truststore.password: 'password'
+ *     ssl.truststore.type: 'JKS'
+ * </pre>
+ */
+public class KafkaHelper {
+
+    /**
+     * Is kafka enabled in configuration 'kafka.enabled' property
+     * @param config configuration
+     * @return true if is enabled or false if not
+     */
+    public static boolean isKafkaEnabled(JsonObject config) {
+        return ConfigUtil.getBoolean("/kafka/enabled", config);
+    }
+
+    /**
+     * Creates {@link KafkaConsumer} from config object
+     * @param vertx vertx instance
+     * @param consumerName consumer name in config
+     * @param config configuration
+     * @param <K> key type
+     * @param <V> value type
+     * @return {@link KafkaConsumer} instance
+     */
+    public static <K, V> KafkaConsumer<K, V> createConsumer(Vertx vertx, String consumerName, JsonObject config) {
+
+        JsonObject kafkaConfig = kafkaConfig(config);
+
+        Map<String, String> commonKafkaConfig = commonKafkaConfig(kafkaConfig);
+        Map<String, String> consumerConfig = consumerConfig(consumerName, kafkaConfig);
+
+        consumerConfig.putAll(commonKafkaConfig);
+
+        return KafkaConsumer.create(vertx, consumerConfig);
+    }
+
+    /**
+     * Creates {@link KafkaProducer} from config object
+     * @param vertx vertx instance
+     * @param producerName consumer name in config
+     * @param config configuration
+     * @param <K> key type
+     * @param <V> value type
+     * @return {@link KafkaConsumer} instance
+     */
+    public static <K, V> KafkaProducer<K, V> createProducer(Vertx vertx, String producerName, JsonObject config) {
+
+        JsonObject kafkaConfig = kafkaConfig(config);
+
+        Map<String, String> commonKafkaConfig = commonKafkaConfig(kafkaConfig);
+        Map<String, String> producerConfig = producerConfig(producerName, kafkaConfig);
+
+        producerConfig.putAll(commonKafkaConfig);
```

Comment:
The same question/note as for consumer above.

Reply:
The same answer as above